android/src/main/java/com/github/yamill/orientation/listeners/OrientationListener.kt

```diff
@@ -0,0 +1,38 @@
+package com.github.yamill.orientation.listeners
+
+import android.app.Activity
+import android.util.Log
+import com.facebook.common.logging.FLog
+import android.view.OrientationEventListener
+import com.facebook.react.bridge.LifecycleEventListener
+import com.facebook.react.bridge.ReactContext
+import com.facebook.react.common.ReactConstants
+
+class OrientationListener internal constructor(
+        private val reactContext: ReactContext,
+        private val onGetCurrentActivity: () -> Activity?,
+) : LifecycleEventListener {
+
+    private lateinit var orientationEventListener: OrientationEventListener;
+
+    override fun onHostResume() {
+        val activity = onGetCurrentActivity()
+        if (activity != null && !this::orientationEventListener.isInitialized) {
+            orientationEventListener = object: OrientationEventListener(activity) {
+                override fun onOrientationChanged(orientation: Int) {
```

Comment:
note: this orientation represents the degrees. It's supposed to range from 0 to 359, but in practice, it is sometimes -1. https://developer.android.com/reference/android/view/OrientationEventListener#onOrientationChanged(int)

Reply:
this might be worth a comment